src/hotspot/share/oops/instanceKlassFlags.hpp

```diff
@@ -49,7 +49,10 @@ class InstanceKlassFlags {
     flag(is_shared_platform_class           , 1 << 8) /* defining class loader is platform class loader */ \
     flag(is_shared_app_class                , 1 << 9) /* defining class loader is app class loader */ \
     flag(has_contended_annotations          , 1 << 10) /* has @Contended annotation */ \
-    flag(has_localvariable_table            , 1 << 11) /* has localvariable information */
```

Comment:
I suggest, as a matter of course, keeping the final `\<newline>` at the end of the list, and then adding something like `/*end of list*/` at the end.  This helps prevent churn on the final line, in such lists.

Reply:
That should be `<backslash><newline>`, in case it didn’t come through markdown.  Removing the final backslash, like removing the final comma an an `enum`, causes churn.  Adding a final comment also makes it easy to visually locate the end of the sequence; it is much clearer than the blank line required by the cpp macro mechanism.

Reply:
Thanks John, that was a good suggestion.  Yes, I'm clearing out the AccessFlags in lots of small changes.